src/components/SuggestionsList/SuggestionsListItem.css

```diff
@@ -6,6 +6,13 @@
   padding: 0.4rem 1.2rem 0.8rem 1.2rem;
 }
 
+.y-suggestion-list-item--badge {
+  background-color: $badgeColor__background;
+  fill: $badgeColor__fill;
+  display: block;
+  top: 0;
```

Comment:
if YamUI Icon always needs to be adjusted here, we should add a rule in Avatar to handle it instead:

```
.y-avatar--badgeContent > .y-icon {
  top: 0;
}
```

not ideal, but better than a one-off for SuggestionsList's Flexternal case